# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsHttpActivityDistributor.cpp
# Commit: a92c423736e5
# Full Hash: a92c423736e5979a8cf039017fd707d41f665b9c
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-05-25 15:42:48
# Regressor Bug: 1546358
# File Overlap Count: 1
# Description:
#   Bug 1546358 - Forward cache entry check to parent process r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75941
# ==============================================================================

diff -r b21cc935a336 -r a92c423736e5 netwerk/protocol/http/nsHttpActivityDistributor.cpp
--- a/netwerk/protocol/http/nsHttpActivityDistributor.cpp	Fri May 22 13:37:54 2020 +0000
+++ b/netwerk/protocol/http/nsHttpActivityDistributor.cpp	Mon May 25 11:07:07 2020 +0000
@@ -8,8 +8,10 @@
 #include "mozilla/net/SocketProcessChild.h"
 #include "mozilla/net/SocketProcessParent.h"
 #include "nsHttpActivityDistributor.h"
+#include "nsHttpHandler.h"
 #include "nsCOMPtr.h"
 #include "nsIOService.h"
+#include "nsQueryObject.h"
 #include "nsThreadUtils.h"
 #include "NullHttpChannel.h"
 
